tests/integration/security/testdata/authz/v1beta1-tcp.yaml.tmpl

```diff
@@ -1,86 +1,86 @@
-# The following policy denies request with path "/data" to port 8090 for workload b
+# The following policy denies request with path "/data" to port 8091 for workload
 
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
-  name: policy-b-deny
+  name: policy-dst0-deny
```

Comment:
I feel the `dst` and `src` naming is just confusing and too hard to read, can we just use the `a`, `b`, ... , etc. in the name and selector? 

you can have `policy-d-deny` with selector `app: {{ .b }}`, it does not prevent you to pass a vm workload for b at all.

And I think we actually should do this for other tests for a better readability in the test code and policy.

Reply:
Can we give another name instead of `a`, `b`,...? The purpose of templatizing the policies is to make it workload agnostic, dst0 is not limited to workload b. Maybe change `policy-b-deny` to `deny-by-port-and-path`?

Reply:
`a`, `b`, ... can be generic just as `dst0`, you can assign the vm to `a` without any problems. `a`, `b`, ... is just a symbol that happens to be the same as the current k8s workloads, which gives the benefit to make sure we're not reducing test coverage and easier to compare the diff.

Reply:
I have replaced dst and src with `a`, `b`, `c`... and updated the workload mapping in the PR description. Only x` is replaced with `e`, others are only parameterized, e.g. `a`->`{{. a }}`. I think this change reduces the diff and makes it easier for review.